### DISCUSSION TITLE
Atomically pick a string_value value if not provided.

### DISCUSSION THREAD

**Discussion by wyatt-herkamp** [SeaQL/sea-orm#2125] [Ideas]
Take the following enum. I think we should automatically use the Variant name if the string_value is not provided. 
```rust
#[derive(DeriveActiveEnum)]
#[sea_orm(rs_type = "String", db_type = "Enum", enum_name = "Interval")]
pub enum Intervals {
    #[sea_orm(string_value = "Weekly")]
    Weekly,
    #[sea_orm(string_value = "BIWeekly")]
    BIWeekly,
    #[sea_orm(string_value = "Monthly")]
    Monthly,
    #[sea_orm(string_value = "Quarterly")]
    Quarterly,
    #[sea_orm(string_value = "Yearly")]
    Yearly,
}
```

This would allow people to do
```rust
#[derive(DeriveActiveEnum)]
#[sea_orm(rs_type = "String", db_type = "Enum", enum_name = "Interval")]
pub enum Intervals {
    Weekly,
    BIWeekly,
    Monthly,
    Quarterly,
    Yearly,
}
```

And get the same result. I would be happy to make a PR if the developers of the project is Ok with this feature.

**Comment by tyt2y3** [comment]
I think it's a good idea to have! However this is no small task, because once we are doing this, we'd at least want to support serde like attributes, e.g. `rename_all = "snake_case"`. But of course if you are willing to undertake such a task, it'd be greatly appreciated!

**Comment by anshap1719** [comment]
@tyt2y3 This sounds like fun work. I'll try and see if I can create a PR :)

Unless @wyatt-herkamp wants to do it, ofcourse

**Comment by wyatt-herkamp** [comment]
> @tyt2y3 This sounds like fun work. I'll try and see if I can create a PR :)
> 
> Unless @wyatt-herkamp wants to do it, ofcourse

Feel free to work on it!. I have been busy with school and trying to fix bugs in Rust Analyzer. If you need help feel free to reach out.

**Comment by anshap1719** [comment]
@wyatt-herkamp The change has now been merged and should be available in the next release.
